programs/github

```diff
@@ -37,8 +92,21 @@ if action == 'clone' then
 
 	dest = shell.resolve(dest or repo)
 
+	local auth = nil
+
+	if flags.authed then
+		-- if no user is found auth should remain nil.
+		auth = github.Auth.get(flags.authed)
+	end
+
+	if flags.authed and not auth then
+		printError('Unknown User!')
+		print(subcommands['auth'].usage)
```

Comment:
hrm is printing usage necessary here? we're returning the printError on all other cases like this.